### DISCUSSION TITLE
Home Page Featured Image Won't Load

### DISCUSSION THREAD

**Discussion by historianem** [CollectionBuilder/collectionbuilder.github.io#76] [General]
I've tried adding a featured image but it's never loaded, https://historianem.github.io/sample-repository-draft/

In the theme.yml under the Home Page section, I've tried to add it as an objectid, as a relative location of an image in the repository, and as a full url, but the featured image has never loaded on the actual site. I've uploaded the picture I'd like to display into the _data folder, but that didn't seem to change anything. Any help would be appreciated, thanks! Loving collection builder so far.

**Comment by evanwill** [comment]
@historianem the issue is the location of the image you added--
the image "panam_tokyotosf_03.jpeg" should not be in the "_data" folder, please delete that copy (the data folder does not directly become part of the final website). 
It looks like you already have a copy of "panam_tokyotosf_03.jpeg" in the "objects" folder, which is the correct location. You can use that in your "theme.yml" like this:

```
featured-image: /objects/panam_tokyotosf_03.jpeg 
```

That should get it working--let me know!

**Reply by evanwill** [reply]
@historianem one more note: I just noticed that you have an item in the collection using that image, so it would probably be better to use the associated objectid instead. This option:

```
featured-image: coll003
```

will add the correct image and provide a link to the item in the collection.

**Reply by historianem** [reply]
I added coll003 to the featured-image section and it worked! Thanks so much for your help.

**Reply by evanwill** [reply]
Great! Looks like a really interesting collection!
Let us know if you have any questions about using in course or anything!
